README.md

```diff
@@ -73,6 +73,12 @@ options for the offset in the file/text provided:
 ]
 ```
 
+To use the iOS SDK is to specified the `compilerargs` option. The value
```

Comment:
Quotes is required.

If you run without quotes It is an command error(`Illegal instruction:4`).
It because the shell will run the argument parsing was delimiter whitespace.